tests/integration/targets/crashfiles_health/tasks/main.yml

```
@@ -0,0 +1,27 @@
+---
+- name: Starting Crash Files health check integration tests
+  ansible.builtin.debug:
+    msg: "START network.healthchecks.crash_files health check integration tests on connection={{ ansible_connection }}"
+
+- name: Run network.crash_files validated content operation
+  block:
+    - name: Run network.crash_files validated content with crash_files health check operation
+      ansible.builtin.include_role:
+        name: network.healthchecks.crashfiles
+      vars:
+        details: true
+        checks:
+          - name: crash_files
+            ignore_errors: true
+            details: true
+            crashinfo_files:
+              - /var/crashinfo
+              - /var/crash
+            cores:
+              - /var/core
+              - /var/crash
```

We need to align these ased on the discussion we had for roles like CPU and memory
```suggestion
       vars:
    crashinfo_files:
      - /var/crashinfo
      - /var/crash
    cores:
      - /var/core
      - /var/crash
    ignore_errors: true
    details: true

```